DragGAN/stylegan_onnx_demo.py

```diff
@@ -0,0 +1,81 @@
+import os
```

Comment:
Is it possible to add some comments what the demo is about.

Reply:
done.